app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Avatar, Tooltip, Dropdown } from "neetoui";
+
+const Note = ({ note }) => (
+  <div className="mb-3 box-border w-full border border-gray-300 bg-white p-4 shadow-sm">
+    <div className="flex justify-between">
+      <Typography style="h4">{note.title}</Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical} position="bottom-end">
+        <li>Edit</li>
+        <li>Delete</li>
+      </Dropdown>
+    </div>
+    <div className="mt-1 mb-3" style={{ color: "#68737D" }}>
+      <Typography style="body2">{note.desc}</Typography>
+    </div>
+    <hr></hr>
+
```

Comment:
No need to add blank lines in JSX. It's visually not helpful but rather makes it more confusing given that this is the presentation logic.

PS: I am talking about JSX here. Not JavaScript code where blank lines can make sense visually.